tools/release-scripts/github.sc

```diff
@@ -0,0 +1,102 @@
+import $ivy.`com.softwaremill.sttp::core:1.1.12`
+import $ivy.`com.typesafe.play::play-json:2.6.9`
+import com.softwaremill.sttp._
+import play.api.libs.json._
+import $file.context
+
+
+
+import scala.concurrent._
+
+
+class GithubClient(token: String)(implicit buildContext: context.BuildContext) {
```

Comment:
We already have a `./ci/githubClient.sc`. What is the motivation for the duplication?